packages/dpkg.rb

```diff
@@ -3,20 +3,20 @@
 class Dpkg < Package
   description 'A medium-level package manager for Debian'
   homepage 'https://anonscm.debian.org/git/dpkg/'
-  version '1.18.25'
-  source_url 'https://salsa.debian.org/dpkg-team/dpkg/-/archive/1.18.25/dpkg-1.18.25.tar.gz'
-  source_sha256 '93470770161bb15fd7988da1890aecda322e19c3fc858d6cb8f33417e5fe01e9'
+  version '1.19.1'
+  source_url 'http://http.debian.net/debian/pool/main/d/dpkg/dpkg_1.19.1.tar.xz'
```

Comment:
What is wrong with `https://salsa.debian.org/dpkg-team/dpkg/-/archive/1.19.1/dpkg-1.19.1.tar.gz`?

Reply:
REmember the fact that we had tio use git before? This source tarball has a configure script.